package-kubeapps-version.sh

```diff
@@ -53,8 +53,6 @@ main() {
   # Generate the package yaml for the staging release first.
   generate_package_yaml "$version" "$packaging_version_suffix" "$version_dir" "$yaml_schema" "$oci_repo"
 
-  # TODO(minelson): Eventually get the sha from the bundle lock to put in the
-  # package.yaml rather than the tag.
```

Comment:
Just removing this as I'd closed the [related issue](https://github.com/vmware-tanzu/package-for-kubeapps/issues/8) after discovering that a tag is completely fine in the package.yaml.